staging/src/k8s.io/client-go/Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/client-go",
-	"GoVersion": "go1.7",
-	"GodepVersion": "v74",
+	"GoVersion": "go1.8",
+	"GodepVersion": "v79",
```

Comment:
update-staging-client-go does godeppy things, but it looks like it tries to force a specific godep version. @caesarxuchao, any tips here?